action.yml

```diff
@@ -21,5 +21,5 @@ outputs:
           description: 'Set to true if the pushed version is a prerelease. otherwise false.'
   
 runs:
-  using: 'node12'
+  using: 'node16'
```

Comment:
```suggestion
  using: 'node20'
```

Reply:
https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/